tubular/strings.py

```diff
@@ -135,3 +135,56 @@ def transform(self, X):
         )
 
         return X
+
+
+class StringConcatenator(BaseTransformer):
+    """
+    Transformer to concatenate values from a few columns into 1 string.
```

Comment:
Sorry the first comment (repeated below) should have been about the "concatenate values from a few columns" in this docstring - all the others reference this with "again"

This feels a bit vague - not immediately clear that values can be any datatype, maybe:

Transform to combine data from specified columns, of mixed datatypes, into a new column containing one string?

(will this work on e.g. a column of dictionaries?)

Reply:
[7ade9ba](https://github.com/lvgig/tubular/pull/52/commits/7ade9ba12014db183180f7b0450691ed7867c468)